### PR TITLE
Docker scripts

### PR DESCRIPTION
**Pull request by gardster**
All necessary scripts to build docker image with both postgres and web part of the Juno Reverse Geocoder

### CODE REVIEW

File: docker/init.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+for i in `seq 1 120`; do
+  echo "Waiting for PostgreSQL start, attempt $i..."
+  ${psql[@]} -c 'select 1;' && break
+  sleep 1
+done
+
+${psql[@]} -c "create extension postgis"
+
+osm2pgsql --slim --drop --number-processes 4 --style=/usr/src/jrg/sql/geocoder.style -U postgres -d postgres --prefix geocoder $1
```
Comment (Zverik) [inline docker/init.sh]: `$JRG_SRC`, and split into two lines plz.

File: docker/init.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+su postgres -c "vacuumdb --full --freeze --analyze --all"
+
+su postgres -c "pg_ctl -D "$PGDATA" -w stop"
+
+su postgres -c "pg_ctl -D \"$PGDATA\" \
+     -o \"-c listen_addresses='' -c checkpoint_completion_target=0.1 -c wal_buffers=32kB\" \
+     -w start"
+
+"${psql[@]}" --username postgres -c "checkpoint;"
+sleep 5;
```
Comment (Zverik) [inline docker/init.sh]: redundant semicolon

File: docker/init.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  echo "Waiting for PostgreSQL start, attempt $i..."
+  ${psql[@]} -c 'select 1;' && break
+  sleep 1
+done
+
+${psql[@]} -c "create extension postgis"
+
+osm2pgsql --slim --drop --number-processes 4 --style=/usr/src/jrg/sql/geocoder.style -U postgres -d postgres --prefix geocoder $1
+
+
```
Comment (Zverik) [inline docker/init.sh:26]: extra newline

File: docker/init.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+psql=( psql -U postgres -v ON_ERROR_STOP=1 )
+
+for i in `seq 1 120`; do
+  echo "Waiting for PostgreSQL start, attempt $i..."
+  ${psql[@]} -c 'select 1;' && break
+  sleep 1
+done
+
+${psql[@]} -c "create extension postgis"
```
Comment (Zverik) [inline docker/init.sh]: Please add a semicolon to make it consistent with queries on lines 18 and 40.

**Review by Zverik — CHANGES_REQUESTED** [review]

File: docker/init.sh (hunk 4 above)
Comment (gardster) [reply docker/init.sh]: Ok

File: docker/init.sh (hunk 1 above)
Comment (gardster) [reply docker/init.sh]: Good catch! Fixed.

**Review by Zverik — APPROVED** [review]

**Pull request merged**